config/converter.go

```diff
@@ -30,7 +30,7 @@ var DefaultCommon = Common{
 
 var DefaultConfigInterface = ConverterConfig{
 	OutputFile:        "./generated/generated.go",
-	OutputPackageName: "generated",
+	OutputPackageName: "", // will try to use existing package name, if can be found
```

Comment:
I think the comment can be omitted, most of the other config values we be filled in later, so we don't have to have a special comment for package name here.